composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/SpotOnField.kt

```diff
@@ -19,7 +19,6 @@ fun SpotOnField(
 ) {
     Box(
         modifier = modifier
-            .height(55.dp)
```

Comment:
Seems like this messed up the configuration of the "Book Now" button in SpotDetailView, will need to pass the modifier with height 55.dp preconfigured.

Reply:
Fixed spacing here, and updated tab view to work properly (there was some weird spacing going on).